docs/src/components/home/sections/ThemingSection.tsx

```diff
@@ -13,6 +14,9 @@ export const ThemingSection = ({ colorMode, platform }) => {
     freezeOnceVisible: true,
   });
   const isVisible = !!entry?.isIntersecting;
+  if (isVisible) {
+    trackScroll('Home#Theming');
```

Comment:
Won't this fire multiple times if there's more than one render and isVisible is true?

Reply:
![CleanShot 2022-06-30 at 13 40 59](https://user-images.githubusercontent.com/321279/176774707-18c24836-d488-44b9-8e85-2f2b589e7ef3.gif)